packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/TeamPromptResponseSummaryCard.tsx

```diff
@@ -8,39 +8,43 @@ import {PALETTE} from '~/styles/paletteV3'
 import {createEditorExtensions} from '../../../../../components/promptResponse/tiptapConfig'
 
 const responseSummaryCardStyles: React.CSSProperties = {
-  margin: '12px',
-  width: '251px'
+  display: 'inherit',
+  padding: '12px',
+  width: '100%'
 }
 
-const promptResponseStyles = {
+const promptResponseStyles: React.CSSProperties = {
   minHeight: '40px',
   lineHeight: '20px',
   border: 'solid',
   borderWidth: '1px',
   borderColor: PALETTE.SLATE_300,
   borderRadius: '4px',
-  padding: '12px 16px 12px 16px'
+  padding: '12px 16px 12px 16px',
+  overflow: 'auto',
+  wordWrap: 'break-word'
```

Comment:
this will make sure to wrap strings with no whitespace, ie links. overflow auto will make sure it'll scroll if there's no enough space